src/sidebar/components/test/annotation-omega-test.js

```diff
@@ -51,20 +52,66 @@ describe('AnnotationOmega', () => {
     $imports.$restore();
   });
 
-  it('renders quote if annotation has a quote', () => {
-    fakeQuote.returns('quote');
-    const wrapper = createComponent();
+  describe('annotation quote', () => {
+    it('renders quote if annotation has a quote', () => {
+      fakeQuote.returns('quote');
+      const wrapper = createComponent();
 
-    const quote = wrapper.find('AnnotationQuote');
-    assert.isTrue(quote.exists());
+      const quote = wrapper.find('AnnotationQuote');
+      assert.isTrue(quote.exists());
+    });
+
+    it('does not render quote if annotation does not have a quote', () => {
+      fakeQuote.returns(null);
+
+      const wrapper = createComponent();
+
+      const quote = wrapper.find('AnnotationQuote');
+      assert.isFalse(quote.exists());
+    });
+  });
+
+  describe('annotation body and excerpt', () => {
+    it('updates annotation state when text edited', () => {
+      const wrapper = createComponent();
+      const body = wrapper.find('AnnotationBody');
+
+      act(() => {
+        body.props().onEditText({ text: 'updated text' });
+      });
+
+      assert.calledOnce(fakeStore.createDraft);
+    });
   });
 
-  it('does not render quote if annotation does not have a quote', () => {
-    fakeQuote.returns(null);
+  describe('tags', () => {
+    it('renders tag editor if `isEditing', () => {
+      // The presence of a draft will make `isEditing` `true`
+      fakeStore.getDraft.returns(fixtures.defaultDraft());
+
+      const wrapper = createComponent();
+
+      assert.isOk(wrapper.find('TagEditor').exists());
+      assert.notOk(wrapper.find('TagList').exists());
+    });
+
+    it('updates annotation state if tags changed', () => {
```

Comment:
```suggestion
    it('updates draft if tags changed', () => {
```

Reply:
I'm rewriting a bunch of tests in my follow-on PR to this one and these descriptions are already updated there!